bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/PowerMaxBinding.java

```
@@ -0,0 +1,907 @@
+                    motionOffDelay = DEFAULT_MOTION_OFF_DELAY;
+                    logger.warn("PowerMax alarm binding: motion off delay not configured correctly (number expected)");
+                }
+            }
+
+            if (StringUtils.isNotBlank((String) config.get("allowArming"))) {
+                allowArming = Boolean.valueOf((String) config.get("allowArming"));
+            }
+            if (StringUtils.isNotBlank((String) config.get("allowDisarming"))) {
+                allowDisarming = Boolean.valueOf((String) config.get("allowDisarming"));
+            }
+            if (StringUtils.isNotBlank((String) config.get("forceStandardMode"))) {
+                forceStandardMode = Boolean.valueOf((String) config.get("forceStandardMode"));
+                PowerMaxReceiveType.POWERLINK.setHandlerClass(
+                        forceStandardMode ? PowerMaxBaseMessage.class : PowerMaxPowerlinkMessage.class);
+            }
+
+            if (StringUtils.isNotBlank((String) config.get("panelType"))) {
+                try {
+                    panelType = PowerMaxPanelType.fromLabel((String) config.get("panelType"));
+                } catch (IllegalArgumentException exception) {
+                    panelType = PowerMaxPanelType.POWERMAX_PRO;
+                    logger.warn("PowerMax alarm binding: panel type not configured correctly");
+                }
+            }
+            PowerMaxPanelSettings.initPanelSettings(panelType);
+
+            if (StringUtils.isNotBlank((String) config.get("autoSyncTime"))) {
+                autoSyncTime = Boolean.valueOf((String) config.get("autoSyncTime"));
+            }
+
+            if (StringUtils.isNotBlank((String) config.get("pinCode"))) {
+                pinCode = (String) config.get("pinCode");
+            }
+
+            properlyConfigured = true;
+
+        } catch (ConfigurationException e) {
+
+            logger.warn("PowerMax alarm binding: {}", e.getReason());
```

1. you would want to rethrow the exception if you caught it, and 
2. perhaps no reason to catch it because it would be logged outside this method anyway?